play2-oauth2-provider/src/main/scala/scalaoauth2/provider/OAuth2ProviderActionBuilders.scala

```diff
@@ -1,19 +0,0 @@
-package scalaoauth2.provider
-
-import play.api.mvc._
-
-import scala.concurrent.{ Future, ExecutionContext }
-
-trait OAuth2ProviderActionBuilders {
```

Comment:
@tsuyoshizawa Hi, what was this replaced with?

Reply:
@Dinduks We separated core and Play project.
https://github.com/nulab/play2-oauth2-provider/blob/master/src/main/scala/scalaoauth2/provider/OAuth2ProviderActionBuilders.scala#L5

See https://github.com/nulab/play2-oauth2-provider